datasets/modeled/transport/dimension_mappings/subsector_to_subsector.csv

```diff
@@ -0,0 +1,721 @@
+from_id,to_id
+Single_Driver+Low_Income+Urban+Compact+BEV_100,bev_compact
```

Comment:
You have 52 subsectors in your data and that don’t seem to match these.

```
+----------------------------------------+                                      
|subsector                               |
+----------------------------------------+
|Personal_LDV+Midsize+BEV_200            |
|Truck_Medium+Local+BEV_150              |
|Truck_Light_Vocational+Regional+BEV_300 |
|Personal_LDV+Midsize+BEV_400            |
|Personal_LDV+Pickup+BEV_300             |
|Personal_LDV+Pickup+BEV_400             |
|Personal_LDV+SUV+PHEV_25                |
|Personal_LDV+SUV+BEV_400                |
|Personal_LDV+Compact+BEV_200            |
|Personal_LDV+Compact+BEV_300            |
|Personal_LDV+Compact+BEV_400            |
|Personal_LDV+SUV+BEV_200                |
|Truck_Heavy+Regional+BEV_300            |
|Personal_LDV+SUV+PHEV_50                |
|Truck_Heavy+Long-haul+BEV_500           |
|Personal_LDV+Compact+PHEV_50            |
|Truck_Medium+Long-haul+BEV_300          |
|Truck_Medium+Long-haul+BEV_500          |
|Truck_Heavy_Vocational+Local+BEV_150    |
|Truck_Heavy+Regional+BEV_150            |
|Personal_LDV+Pickup+PHEV_25             |
|Personal_LDV+Pickup+PHEV_50             |
|Personal_LDV+Midsize+PHEV_50            |
|Truck_Light+Long-haul+BEV_300           |
|Truck_Light+Local+BEV_300               |
|Truck_Light_Vocational+Local+BEV_150    |
|Truck_Light_Vocational+Local+BEV_300    |
|Truck_Light+Local+BEV_150               |
|Truck_Medium+Regional+BEV_150           |
|Truck_Medium_Vocational+Regional+BEV_300|
|Truck_Light_Vocational+Regional+BEV_150 |
|Bus+N/A+BEV_200                         |
|Truck_Heavy+Local+BEV_150               |
|Truck_Heavy+Local+BEV_300               |
|Personal_LDV+SUV+BEV_300                |
|Truck_Heavy_Vocational+Local+BEV_300    |
|Truck_Heavy_Vocational+Regional+BEV_150 |
|Personal_LDV+Pickup+BEV_200             |
|Personal_LDV+Midsize+BEV_300            |
|Truck_Light+Regional+BEV_300            |
|Truck_Medium+Regional+BEV_300           |
|Truck_Heavy+Long-haul+BEV_300           |
|Truck_Light+Long-haul+BEV_500           |
|Truck_Light+Regional+BEV_150            |
|Truck_Heavy_Vocational+Regional+BEV_300 |
|Truck_Medium+Local+BEV_300              |
|Personal_LDV+Compact+PHEV_25            |
|Truck_Medium_Vocational+Regional+BEV_150|
|Transit_Rail                            |
|Personal_LDV+Midsize+PHEV_25            |
|Truck_Medium_Vocational+Local+BEV_150   |
|Truck_Medium_Vocational+Local+BEV_300   |
+----------------------------------------+
```

Reply:
thanks, fixed